iceoryx_posh/test/moduletests/test_roudi_memory_manager.cpp

```diff
@@ -108,4 +108,7 @@ TEST_F(RouDiMemoryManager_Test, AddMemoryProviderExceedsCapacity)
     ASSERT_THAT(expectError.has_error(), Eq(true));
     EXPECT_THAT(expectError.get_error(), Eq(RouDiMemoryManagerError::MEMORY_PROVIDER_EXHAUSTED));
 }
+
+/// @todo Add TEST_F(RouDiMemoryManager_Test, StreamOperatorTranslationIsCorrect)
```

Comment:
Could you please add this todo into an issue as an open point?

Reply:
Done: https://github.com/eclipse/iceoryx/issues/91#issuecomment-746314090